Automattic/woocommerce-payments
# Enable transaction dispute details feature for all merchants (remove feature flag)

Fixes #7289

#### Changes proposed in this Pull Request

This PR enables the transaction dispute details feature for all merchants by removing the feature flag `_wcpay_feature_dispute_on_transaction_page`.

#### TODO:
- [x] Create changelog entry
	`Display dispute information, recommended resolution steps, and actions directly on the transaction details screen to help merchants with dispute resolution.`

Post-merge:
- [x] [Update any affected critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows)
- [x] [Create release testing instructions](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions): https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions-for-WC-Payments-6.6.0#enable-transaction-dispute-details-feature-for-all-merchants-pr-7365

**Before**
<img width="1057" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/46842146-0cce-4b63-897f-c15fffdf6b06">

**After**
<img width="1042" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/b5b5ec45-2f37-4036-b5c9-a65a3c879092">



#### Testing instructions





* Ensure you've disabled the feature flag `_wcpay_feature_dispute_on_transaction_page` if previously enabled
	* `delete_option( '_wcpay_feature_dispute_on_transaction_page' )`
* Visit a dispute details screen and ensure it redirects to the transaction details screen
* Ensure the dispute details section appears on the transaction details screen for
	* Dispute awaiting a response
	* Dispute under review
	* Dispute lost
	* Dispute won
	* Inquiry awaiting a response
	* Inquiry under review
	* Inquiry closed

-------------------


**Post merge**



- [x] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions-for-WC-Payments-6.6.0#enable-transaction-dispute-details-feature-for-all-merchants-pr-7365
